cmd/runtime/main.go

```diff
@@ -53,9 +56,28 @@ func run() error {
 		}
 	}()
 
-	l, err := net.Listen("unix", socketPath)
+	var (
+		network string
+		address string
+	)
+
+	if socketPath == "" {
+		network = "unix"
+		address = socketPath
+
+		if err := os.Remove(socketPath); err != nil {
+			if !errors.Is(err, os.ErrNotExist) {
+				log.Fatal(err)
```

Comment:
I think you should return error here instead of panicking.

Reply:
Whoops